pyvista/core/pointset.py

```diff
@@ -766,6 +766,34 @@ def faces(self, faces):
             # TODO: faster to mutate in-place if array is same size?
             self.SetPolys(CellArray(faces))
 
+    @property
+    def strips(self) -> np.ndarray:
+        """Return a pointer to the strips as a numpy array.
+
+        Returns
+        -------
+        numpy.ndarray
+            Array of strip indices.
+
+        Examples
+        --------
+        >>> import pyvista as pv
+        >>> polygon = pv.Rectangle()
+        >>> extruded = polygon.extrude((0, 0, 1))
+        >>> extruded.strips
+        array([4, 0, 1, 4, 5, 4, 1, 2, 5, 6, 4, 2, 3, 6, 7, 4, 3, 0, 7, 4],
+              dtype=int64)
```

Comment:
The doctest isn't happy with the formatting of this output

Reply:
```
_______________ [doctest] pyvista.core.pointset.PolyData.strips ________________
774         -------
775         numpy.ndarray
776             Array of strip indices.
777 
778         Examples
779         --------
780         >>> import pyvista as pv
781         >>> polygon = pv.Rectangle()
782         >>> extruded = polygon.extrude((0, 0, 1))
783         >>> extruded.strips
Expected:
    array([4, 0, 1, 4, 5, 4, 1, 2, 5, 6, 4, 2, 3, 6, 7, 4, 3, 0, 7, 4],
          dtype=int64)
Got:
    array([4, 0, 1, 4, 5, 4, 1, 2, 5, 6, 4, 2, 3, 6, 7, 4, 3, 0, 7, 4])
```

Reply:
This is probably a windows vs "everything else" thing. On windows int64 is additional info, gets printed. Everywhere else int64 is the default.

Reply:
I'm working on a windows machine, I just copied the output from the python output and the test passed in my local machine. I removed that `int64` in commit [a644a86](https://github.com/pyvista/pyvista/pull/3208/commits/a644a86454cf6c8e68ee3c09e13d89faaebd9800).